CMakeLists.txt

```diff
@@ -600,6 +600,18 @@ else()
   message(STATUS "JNI library is disabled")
 endif()
 
+# Installation and packaging for Linux
+include(GNUInstallDirs)
+install(TARGETS ${ROCKSDB_STATIC_LIB} COMPONENT devel ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR})
+install(TARGETS ${ROCKSDB_SHARED_LIB} COMPONENT runtime DESTINATION ${CMAKE_INSTALL_LIBDIR})
+install(DIRECTORY include/rocksdb COMPONENT devel DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
+if(${CMAKE_INSTALL_PREFIX} STREQUAL /usr/local)
```

Comment:
I'm as a user expect libraries to be installed to `/usr/local` when I do `cmake -DCMAKE_INSTALL_PREFIX=/usr/local/`. With this code behave is quite surprising for me - it will be installed to `/usr` instead even if I explicitly set another destination. 

In 99% cases the use of CMAKE_INSTALL_PREFIX in CMakeLists.txt means you're doing something wrong.